_posts/2022-05-17-calldata-reencode-size-check-bug.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+published: true
+title: 'Size check on ABI-reencoding nested calldata arrays bug'
```

Comment:
That is a very long title :) Can we shorten it? If not, we at least need to capitalize the first letter of each word.

Reply:
Yeah... it's a very specific bug case... any shortening will make it appear worse than it is... I'll capitalize for now.

Reply:
If we're not dead set on having "bug" at the end, we could also word it like this:
```
Size check bug in nested calldata array ABI-reencoding
```

Reply:
Fine with me either way... that indeed does sound nicer - it's just that such posts *usually* had bug in the end... I'll change it to your suggestion now, though.